tests/governance_root_ca_cert_mismatch.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+import os
+import sys
+import tempfile
+from cryptography import x509
+import cryptography.hazmat.backends as crypto_backends
+import infra.network
+import infra.path
+import infra.proc
+import infra.net
+import infra.e2e_args
+from infra.proposal import ProposalState
+import suite.test_requirements as reqs
+import ccf.proposal_generator
+
+from loguru import logger as LOG
+
+this_dir = os.path.dirname(__file__)
+
+
+@reqs.description("Add certificate with mismatching MRSIGNER")
+def test_cert_mrsigner_mismatch(network, args):
```

Comment:
Can we put this test case in `governance_root_ca_cert.py` rather than its own file? Just add a call to `network = test_cert_mrsigner_mismatch` at the end of `run()`,

Reply:
It needs to be its own file because it uses a different gov script.

Reply:
args are passed explicitly to network.start_and_join(args), this could be in the same file and run against a network passed different args.

Another possibility would be to reuse the network and update the constitution to the "mismatch" variant before proceeding with the test, in which case what Eddy describes could be done.